Docs 5120 landing page for api products pa (#120)
mulesoft/docs-api-manager: modules/ROOT/pages/api-groups-promote-api-instance.adoc

= Promote API Instances in API Groups
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

You can promote an API Group instance from one environment to another, for example, Sandbox to Production.

> Promoting an API Group instance affects only the API definition, not the existing deployed applications subscribed to the API Group. After promoting an API Group instance, the API Name remains the same, but the API Version changes.

Reviewer: I don't know where did you get this, but I think you are describing the promotion for an API instance. Promoting a group instance does not change anything in existing group instances, the only it does is create a group instance from a specified group instance in another environment. The API instances of the promoted group instance may be different than the ones in the original group instance, but the definition (the related assets) remains the same.

Author: Done! Thanks!

> Because each environment has a different client ID and secret, you must update the application’s client ID and secret.

Reviewer: Why is the sentence here? I don't understand why users have to change their credentials (and where)

Author: Removed! Done! Thanks!

> // You cannot prompte a API Group instnace if every API instnace is that group is not present in the Production environment also.

Reviewer: This sentence nearly correct, you must add it. 
```suggestion
// You cannot promote an API Group instance if every API in that group does not have an API instance in any environment of the same environment type that the destination enviroment.
```

Author: Done! Thanks!

//what permission is required for this

To promote an API Group instance:

. From the left menu in API Manager, select *API Groups > Promote from environment*.
. From the *Source Environment* list, select the environment of the API you want to promote.
. From the *API Group* list, select the API Group to which the API Instance belongs.
. From the *API Group version* list, select the version of the API Group to which the instance belongs.
. From the *API Group instance* list, select the API Group instance that you want to promote.

> . Optionally, select the *Import default limits from existent SLA Tiers* checkbox if you want to include the SLA tiers when promoting from source to destination environment.

Reviewer: Only the default limits of every tier will be promoted

Author: Done! Thanks!

= Promote API Instances in API Groups
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

You can promote an API Group instance from one environment to another, for example, sandbox to production. When you promote an API Group instance, the source environment of the instance remains unchanged. However, the destination environment changes by creating the API Group instance that is being promoted. 

> The API instances of the promoted API Group instance may differ from the original group instance, but the definition (related assets) remains the same.

Reviewer 2: may > might, always
```suggestion
The API instances of the promoted API Group instance might differ from the original group instance, but the definition (related assets) remains the same.
```

Author: Done! Thanks!

== Task Prerequisites

Before you begin, ensure that you have:

> * Obtained the API Group Administrator permissions to be able to promote an API Group instance.

Reviewer 2: no period
```suggestion
* Obtained the API Group Administrator permissions to be able to promote an API Group instance
```

Author: Done! Thanks!

> * Verify that the APIs in the API Group instance that you want to promote are already present in the destination environment. For example, if the API Group Instance that you want to promote contains the Facebook API, the destination environment must already have the Facebook API in that environment type.

Reviewer 2: ditto
```suggestion
* Verified that the APIs in the API Group instance that you want to promote are already present in the destination environment +
For example, if the API Group Instance that you want to promote contains the Facebook API, the destination environment must already have the Facebook API in that environment type.
```

Author: Done! Thanks!
